src/unittest_json.nim

```diff
@@ -50,7 +62,11 @@ proc newJsonOutputFormatter*(stream: Stream): JsonOutputFormatter =
 
 proc close*(formatter: JsonOutputFormatter) =
   ## Completes the report and closes the underlying stream.
-  formatter.stream.write($(%formatter.result))
+  let json = %formatter.result
+  for test in json["tests"]:
+    test["test_code"] = test["testCode"]
+    test.delete "testCode"
+  formatter.stream.write($json)
```

Comment:
This is to get around the `--styleCheck:error` issue https://github.com/exercism/nim-test-runner/pull/102#issuecomment-918538360